frontend/internal-packages/agent/src/client.ts

```diff
@@ -0,0 +1,4 @@
+// Client-safe exports that can be used in browser environments
```

Comment:
We may also use the `client-only` package.

- ref: 
    - https://www.npmjs.com/package/client-only
    - https://nextjs.org/docs/app/getting-started/server-and-client-components#preventing-environment-poisoning